lib/console.rb

```diff
@@ -1,21 +1,32 @@
 class Console
 
+  ROW_SEPARATOR = "\n -----------\n"
+
   def initialize(game)
     @game = game 
   end
 
-  def board_display
-    add_padding(@game.board.cells.each_slice(3)
-    .flat_map {|row| [row, "\n"]}
-    .join(" | ")
-    .prepend("| "))
+  def format_board_display
+    rows = create_rows
+    display = ROW_SEPARATOR
+    rows.each do |row|
+       display = draw_row(display, row)
```

Comment:
This seems odd, why are you overwriting display each time? I can't see how this would pass